linux_os/guide/system/permissions/files/dir_perms_world_writable_system_owned_group/oval/shared.xml

```diff
@@ -0,0 +1,22 @@
+<def-group>
+  <definition class="compliance" id="dir_perms_world_writable_system_owned_group" version="1">
+    {{{ oval_metadata("All world writable directories should be group owned by a system user.") }}}
+    <criteria comment="check for local directories that are world writable and have gid greater than or equal to {{{ auid }}}" negate="true">
+      <criterion comment="check for local directories that are world writable and have gid greater than or equal to {{{ auid }}}" test_ref="test_dir_world_writable_gid_gt_value" />
+    </criteria>
+  </definition>
+  <unix:file_test check="all" comment="check for local directories that are world writable and have gid greater than or equal to {{{ auid }}}" id="test_dir_world_writable_gid_gt_value" version="1">
+    <unix:object object_ref="all_local_directories_gid" />
+    <unix:state state_ref="state_gid_is_user_and_world_writable" />
+  </unix:file_test>
+  <unix:file_object comment="all local directories" id="all_local_directories_gid" version="1">
+    <unix:behaviors recurse="directories" recurse_direction="down" max_depth="-1" recurse_file_system="local" />
+    <unix:path operation="equals">/</unix:path>
+    <unix:filename xsi:nil="true" />
+    <filter action="include">state_gid_is_user_and_world_writable</filter>
+  </unix:file_object>
+  <unix:file_state comment="gid greater than or equal to {{{ auid }}} and world writable" id="state_gid_is_user_and_world_writable" version="1">
```

Comment:
Nitpick: `auid` should probably be `uid_min`.

At the moment, by default, `auid` is set to `uid_min` when not explicitly defined.
If one would want to customize the starging range audited users (`auid`), this rule would change behavior too (I admit I haven't heard o such use case).

Reply:
I'm not sure exactly what to decide.

Do you think that the same applies for? https://github.com/ComplianceAsCode/content/pull/6421/files/c11311736558613b13ae051a2908c31eee0b6a43#diff-a840b889a179d9be9756693119366b0e213700be3da9341baa6b5644905f1eccR18

Reply:
Yes.